src/main/java/uk/ac/cam/cl/dtg/isaac/dao/EventBookingPersistenceManager.java

```diff
@@ -255,7 +255,13 @@ private EventBookingDTO convertToDTO(final EventBooking eb, final IsaacEventPage
         try {
             UserSummaryWithEmailAddressDTO user = userManager.convertToDetailedUserSummaryObject(userManager.getUserDTOById(eb
                     .getUserId()), UserSummaryWithEmailAddressDTO.class);
-
+            Long reservedById = eb.getReservedBy();
+            // af599 TODO: Make this null, not 0L...
+            // af599 honestly can't remember the circumstances of this. This *should* come back as null, not 0L...
+            if (!reservedById.equals(0L)) {
```

Comment:
:S 0L is weird. Is postgres set to default to that or null? Is it a foreign key to userId?

Reply:
I forgot to set the default to null. Now I did and re-tested it, and null values still come back as `0L`.